fees/gains-network.ts

```
@@ -79,6 +82,7 @@ const fetchApechain = async (_a: number, _b: ChainBlocks, { createBalances, getL
   stakingFee.forEach((i: any) => dailyHoldersRevenue.add(APE, i.amountCollateral));
   gTokenFee.forEach((i: any) => dailySupplySideRevenue.add(APE, i.amountCollateral));
   referralFee.forEach((i: any) => dailySupplySideRevenue.add(APE, i.amountCollateral));
+  borrowingFee.forEach((i: any) => dailySupplySideRevenue.add(APE, i.amountCollateral));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)